openvdb/openvdb/tools/FastSweeping.h

```diff
@@ -923,8 +930,12 @@ struct FastSweeping<SdfGridT, ExtValueT>::MinMaxKernel
         for (auto leafIter = r.begin(); leafIter; ++leafIter) {
             for (auto voxelIter = leafIter->beginValueOn(); voxelIter; ++voxelIter) {
                 const SdfValueT v = *voxelIter;
-                if (v < mMin) mMin = v;
-                if (v > mMax) mMax = v;
+                const SdfValueT fltMinDif = std::abs(v + std::numeric_limits<SdfValueT>::max());
+                const SdfValueT fltMaxDif = std::abs(v - std::numeric_limits<SdfValueT>::max());
+                if (v < mMin && fltMinDif >= 1.0e-4f) mMin = v;
+                if (v > mMax && fltMaxDif >= 1.0e-4f) mMax = v;
```

Comment:
I'm a bit confused by the 1e-4, I'd expect the fltFooDiff to always saturate to zero?

Reply:
You are right, the problem is not caused by a value that is near float-max. Rather, from a value that is untouched by the sweep algorithm when it was set to float-max originally. I addressed this issue in a new commit.